infinispan/runtime/src/main/java/io/quarkiverse/langchain4j/infinispan/runtime/InfinispanEmbeddingStoreConfig.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkiverse.langchain4j.infinispan.runtime;
+
+import static io.quarkus.runtime.annotations.ConfigPhase.RUN_TIME;
+
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+/**
+ * Configuration of the Redis embedding store.
+ */
+@ConfigRoot(phase = RUN_TIME)
+@ConfigMapping(prefix = "quarkus.langchain4j.infinispan")
+public interface InfinispanEmbeddingStoreConfig {
+
+    /**
+     * The dimension of the embedding vectors. This has to be the same as the dimension of vectors produced by
+     * the embedding model that you use. For example, AllMiniLmL6V2QuantizedEmbeddingModel produces vectors of dimension 384.
+     * OpenAI's text-embedding-ada-002 produces vectors of dimension 1536.
+     */
+    Long dimension();
+
+    /**
+     * Name of the cache that will be used in Infinispan when searching for related embeddings.
+     * If this cache doesn't exist, it will be created.
+     */
+    @WithDefault("lanchain-cache")
```

Comment:
```suggestion
    @WithDefault("langchain-cache")
```

I'm actually wondering if langchain should be part of the name.

Reply:
I'll rename to vector-cache

Reply:
or embeddings-cache